XLS-0066d-lending-protocol/README.md

```diff
@@ -0,0 +1,1560 @@
+| `OwnerNode`               |    `N/A`    | :heavy_check_mark: | `number`  |   `UINT64`    |                        `N/A`                        | Identifies the page where this item is referenced in the `Borrower` owner's directory.                                                                         |
+| `LoanBrokerNode`          |    `N/A`    | :heavy_check_mark: | `number`  |   `UINT64`    |                        `N/A`                        | Identifies the page where this item is referenced in the `LoanBroker`s owner directory.                                                                        |
+| `LoanBrokerID`            |    `No`     | :heavy_check_mark: | `string`  |   `HASH256`   |                        `N/A`                        | The ID of the `LoanBroker` associated with this Loan Instance.                                                                                                 |
+| `Borrower`                |    `No`     | :heavy_check_mark: | `string`  |  `AccountID`  |                        `N/A`                        | The address of the account that is the borrower.                                                                                                               |
+| `LoanOriginationFee`      |    `No`     | :heavy_check_mark: | `number`  |   `NUMBER`    |                        `N/A`                        | A nominal funds amount paid to the `LoanBroker.Owner` when the Loan is created.                                                                                |
+| `LoanServiceFee`          |    `No`     | :heavy_check_mark: | `number`  |   `NUMBER`    |                        `N/A`                        | A nominal funds amount paid to the `LoanBroker.Owner` with every Loan payment.                                                                                 |
+| `LatePaymentFee`          |    `No`     | :heavy_check_mark: | `number`  |   `NUMBER`    |                        `N/A`                        | A nominal funds amount paid to the `LoanBroker.Owner` when a payment is late.                                                                                  |
+| `ClosePaymentFee`         |    `No`     | :heavy_check_mark: | `number`  |   `NUMBER`    |                        `N/A`                        | A nominal funds amount paid to the `LoanBroker.Owner` when a payment full payment is made.                                                                     |
+| `OveraymentFee`           |    `No`     | :heavy_check_mark: | `number`  |   `NUMBER`    |                        `N/A`                        | A fee charged on overpayments in 1/10th basis points. Valid values are between 0 and 100000 inclusive. (0 - 100%)                                              |
```

Comment:
```suggestion
| `OverpaymentFee`           |    `No`     | :heavy_check_mark: | `number`  |   `NUMBER`    |                        `N/A`                        | A fee charged on overpayments in 1/10th basis points. Valid values are between 0 and 100000 inclusive. (0 - 100%)                                              |
```
Or should it be `OverPayment`? I think my suggestion is correct because "overpayment" is a word.